lib/AST/ASTImporter.cpp

```diff
@@ -2100,7 +2152,10 @@ Decl *ASTNodeImporter::VisitEnumDecl(EnumDecl *D) {
   unsigned IDNS = Decl::IDNS_Tag;
   DeclarationName SearchName = Name;
   if (!SearchName && D->getTypedefNameForAnonDecl()) {
-    SearchName = Importer.Import(D->getTypedefNameForAnonDecl()->getDeclName());
+    if (auto Err = ImportOrError(
+        SearchName, D->getTypedefNameForAnonDecl()->getDeclName()))
+      // FIXME: handle error
+      return nullptr;
```

Comment:
Once the visit functions have a return type of `Expected<T>` then this will be `return std::move(Err);` , right?